apps/ai-backoffice/lib/hooks/use-pipeline.ts

```
@@ -0,0 +1,132 @@
+      return
+    }
+
+    // ID to sync return values between multiple ongoing pipe executions
+    let currentId = 0
+
+    const callbacks = new Map<number, (data: PipeReturnType) => void>()
+
+    const onMessageReceived = (e: MessageEvent<OutgoingEventData>) => {
+      switch (e.data.type) {
+        case 'result':
+          const { id, data: serializedData } = e.data
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)